keylime-agent/src/agent_handler.rs

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2023 Keylime Authors
+
+use crate::common::JsonWrapper;
+use crate::{tpm, Error as KeylimeError, QuoteData};
+use actix_web::{web, HttpRequest, HttpResponse, Responder};
+use base64::{engine::general_purpose, Engine as _};
+use log::*;
+use serde::{Deserialize, Serialize};
+use std::{
+    fs::{read, read_to_string},
+    io::{Read, Seek},
+};
```

Comment:
None of these under `std::fs` and `std::io` seems to be used in this file.

Reply:
good catch, removing